lib/csetgrp.gi

```diff
@@ -536,7 +536,7 @@ function(d)
   return(STRINGIFY("DoubleCoset(\<",
```

Comment:
This is indeed weird. I would expect it to start with `\>` not `\<`. More like so
```suggestion
  return(STRINGIFY("DoubleCoset(\>",
                   ViewString(LeftActingGroup(d)),"\<,\>",
                   ViewString(Representative(d)),"\<,\>",
```

Reply:
The point is that I do not know what the idea behind this code was.
Also the fact that whitespace behind the commas is missing is unusual. 
I guess this code was never really used, otherwise the indentation problem would have been noticed earlier.
Shall we just change this method?